Source/utils/endian_int.h

```diff
@@ -0,0 +1,114 @@
+#pragma once
+
+#include <type_traits>
+
+namespace devilution {
+
+template<typename T>
+T SwapLE(T in)
+{
+	static_assert(std::is_integral_v<T> || std::is_enum_v<T>, "unsupported type");
```

Comment:
`std::is_integral_v` and `std::is_enum_v` are not available on older GCC, use `std::is_integral<T>::value` and `std::is_enum<T>::value` instead